packages/airnode-protocol/src/index.ts

```diff
@@ -19,10 +20,11 @@ import {
 } from './contracts';
 import references from '../deployments/references.json';
 
-const AirnodeRrpAddresses: { [chainId: number]: string } = references.AirnodeRrpV0;
-const AccessControlRegistryAddresses: { [chainId: number]: string } = references.AccessControlRegistry;
-const RequesterAuthorizerWithAirnodeAddresses: { [chainId: number]: string } =
+const AirnodeRrpAddresses: { [chainId: string]: string } = references.AirnodeRrpV0;
+const AccessControlRegistryAddresses: { [chainId: string]: string } = references.AccessControlRegistry;
+const RequesterAuthorizerWithAirnodeAddresses: { [chainId: string]: string } =
```

Comment:
I also noticed the types for the keys were set to numbers, but they should actually be strings so changed them here. It doesn't really affect any functionality though.